test/components/Icon.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+
+import Icon from '../../src/Icon';
+
+describe('Icon', () => {
+
+  it('renders correctly', () => {
+    const component = renderer.create(
+      <Icon icon='foo' />
+    );
+    expect(component).toMatchSnapshot();
+  });
+
+  it('renders null if no icon is passed', () => {
+    const component = renderer.create(
+      <Icon />
+    );
+    expect(component.toJSON()).toBeNull();
+  });
+
+  it('renders no child when path is 1', () => {
+    const component = shallow(
```

Comment:
With a small change in the expectation, this could be tested using `renderer.create` instead of `shallow`, but I didn't see any performance improvement, and we cannot remove the `enzyme` dependency because we still need it for the `onClick` test below.